starfish.pro

```diff
@@ -1,5 +1,16 @@
 TEMPLATE = subdirs
-SUBDIRS = starfish starfishd
+
+#CONFIG += utouch
```

Comment:
Maybe we should rename the current starfish.pro into a src.pro and make both telescope and starfish choosable via CONFIG.
starfishd should be built for both platforms anyway. (but it might need a new name now...)